Dockerfile

```diff
@@ -79,6 +79,8 @@ RUN apt-get update && apt-get install -y \
     libxerces-c3.2 \
     libxml2 \
     libzmq5 \
+    python3.9 \
```

Comment:
Why is Python 3.9 explicitly required? Python 3 is already installed in the previous step.

Reply:
The ´libpython` package does not install the python binary. There is neither `python` nor `python3.9` available in the image without `python3.9`.

Reply:
/usr/bin/python3 exists